terraform/modules/aws_cargohold-asset-storage/README.md

```diff
@@ -0,0 +1,20 @@
+Terraform module: Cargohold Asset Storage
+=========================================
+
+This module creates an Object Storage on AWS for cargohold to store encrypted assets.
+
+
+#### How to use the module
+
+```hcl-terraform
+module "gundeck-push-notification" {
+  source = "github.com/wireapp/wire-server-deploy.git//terraform/modules/aws_cargohold-asset-storage?ref=develop"
+  
+  region = "eu-central-1"
+  environment = "staging"
+  account_id = "myAccountID"
+  bucket_name = "my-assets"   # will be prefixed with $environment
```

Comment:
Leave out bucket_name, account_id, and region, as those are not used and/or can have a reasonable default.